app/controllers/application_controller.rb

```diff
@@ -49,4 +50,14 @@ def configure_permitted_parameters
       )
     end
   end
+
+  def check_referal
+    if bad_referal.include?(URI(request.referer).host)
+      flash[:info] = "Did you know The Odin Project is 100% free of charge? #{view_context.link_to('Click Here', faq_path)} to learn more".html_safe
```

Comment:
If this isn't sanitized properly let me know

Reply:
Link via GET to a REST endpoint is A-OK to me 👌